libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -624,8 +641,8 @@ void AP_Airspeed::update(bool log)
 
     check_sensor_failures();
 
-#ifndef HAL_BUILD_AP_PERIPH
-    if (log) {
+#if HAL_LOGGING_ENABLED
+    if (_log_bit != (uint32_t)-1 && AP::logger().should_log(_log_bit)) {
```

Comment:
We do have to be careful to not break logging on periph here; we may actually need to preserve a special-case for AP_Periph if `-1` is to remain "no logging".

Reply:
Is there logging of ARSPD on AP_Periph? From my reading of the code there wasn't?